web/src/components/DescriptionBox/DescriptionBox.tsx

```diff
@@ -0,0 +1,70 @@
+// @ts-nocheck
+
+import * as collapsible from "@zag-js/collapsible";
+import { useMachine, normalizeProps } from "@zag-js/react";
+import React, { useEffect, useState } from "react";
+import './DescriptionBox.css'
+const showdown = require("showdown");
+const showdownKatex = require("showdown-katex")
+
+const getId = (name: string) => `datavzrd-${name}`;
```

Comment:
I am seeing this line for the second time now and there are 3 more identically lines. Can't this be put into a function that can then be used everywhere?

Reply:
I can probably define getId function on App.tsx and export it then import it on all components and use it

Reply:
Yeah that would be great thanks!